src/index.js

```diff
@@ -59,7 +58,6 @@ class TelldusTDToolPlatform {
             `at ${githubRepo}`)
             return null
         }
-
```

Comment:
Please leave this line be, as removing it really hasn't that much to do with the intention of the PR.